docs/visual/faq.md

```diff
@@ -44,6 +44,11 @@ document.body.style.overflow = 'auto'
 ```
 
 
+### What happens if my page has broken HTML tags?
+
+Sauce Visual assumes that there are no broken HTML tags on your pages and will not check for this.
```

Comment:
Hi @eugenekatsov , when I looked at the source page, it said that the only change was the addition of this Q above. I wanted to make sure that was right!

Reply:
yup, correct